assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -435,6 +455,17 @@ function Carousel() {
                 />
               </OverflowButtons>
             )}
+            <SafeModeButton
+              width="24"
+              height="24"
```

Comment:
These static values can be moved to the s-c definition above as:

```js
const SafeModeButton = styled(SafeMode).attrs({
  width: 24,
  height: 24,
})`
```

When/if you fix this, please do the same with `GridViewButton`. Perhaps even using the same object as `.attrs(buttonSize)`.